documentation/content/vane-admin/graylist.md

```diff
@@ -18,6 +18,18 @@ you can assign a new player to the `user` group by executing `perm add player_na
 
 This feature is **disabled** by default, as it often caused confusion.
 
+### Quick start
+
+- Enable in `vane-admin` config: set `world_protection.enabled: true`, then run `/vane reload`.
```

Comment:
Let's only state the configuration that must be changed, and not mention `/vane reload` in this context. I don't want to proliferate any usage of server or plugin config reload (even though the vane-internal command should do the right thing most of the time). Since server reloads are always the reason for subtle bugs in many plugins I think it is better to always recommend restarting the server.